src/appInsights.js

```diff
@@ -0,0 +1,17 @@
+import config from 'config'
+/*
+To collect end-user usage analytics about your application, 
+insert the following script into each page you want to track.
+Place this code immediately before the closing </head> tag,
+and before any other scripts. Your first data will appear 
+automatically in just a few seconds.
+Please find more info here: https://docs.microsoft.com/en-us/azure/application-insights/app-insights-javascript
```

Comment:
http://apmtips.com/blog/2015/03/18/javascript-snippet-explained/ has more detailed info on this particular snippet.

This file can likely be merged into appInsightsService; if it can't, then appInsights can be exported from here instead of appInsightsService.